draft-ietf-tls-tls13.md

```diff
@@ -2159,7 +2176,15 @@ Although X9.62 supports multiple point formats, any given curve
 MUST specify only a single point format. All curves currently
 specified in this document MUST only be used with the uncompressed
 point format (the format for all ECDH functions is considered
-uncompressed).
+uncompressed). Peers MUST validate each other's public value Y by ensuring
+that the point is a valid point on the elliptic
+curve. For all elliptic curves except for x25519 and x448, the appropriate
```

Comment:
Some relevant but not completely fleshed-out bits that might be good to reference:
https://tools.ietf.org/html/rfc7748#section-7

> Designers should also be aware that implementations of these curves
   might not use the Montgomery ladder as specified in this document,
   but could use generic, elliptic-curve libraries instead.  These
   implementations could reject points on the twist and could reject
   non-minimal field elements.  While not recommended, such
   implementations will interoperate with the Montgomery ladder
   specified here but may be trivially distinguishable from it.  For
   example, sending a non-canonical value or a point on the twist may
   cause such implementations to produce an observable error while an
   implementation that follows the design in this text would
   successfully produce a shared key.